Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -185,7 +183,6 @@ void EbrApplicationActivated(IActivatedEventArgs^ args) {
 
 void _ApplicationMainLaunch(ActivationType activationType, Platform::Object^ activationArg) {
     _ApplicationLaunch(activationType, activationArg);
-
     _appEvents = ref new AppEventListener();
     _appEvents->_RegisterEventHandlers();
```

Comment:
We also need the Window to be activated to register event handlers, which we do in _ApplicationLaunch

---
In reply to: [72367773](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72367773) [](ancestors = 72367773,72367429,72352741,72338254)